code-scanning/zscaler.yml

```diff
@@ -0,0 +1,46 @@
+#This workflow uses actions that are not certified by GitHub.
```

Comment:
Can we this file name as "zscaler-iac-scan.yaml"? zscaler.yaml is too generic and there might be other GitHub Actions in future under zscaler.